### ISSUE TITLE
ADB commands cannot be run due to "failed calling service package".

### ISSUE THREAD

**Issue by pthfdr-42** [termux/termux-app#1209]
**Problem description**
A clear and concise description of what the problem with the termux app is. You may post screenshots in addition to description.
To trigger a manual ART recompilation, the command is `adb shell cmd package compile -m speed -f -a`.
The real command without using `adb` is `cmd package compile -m speed -f -a`.
If I run this as root using Magisk's `su` it runs fine.
However, if I use LineageOS' native `su` I get an error.
**Steps to reproduce**
Please post all steps that are needed to reproduce the issue.

    u0_a102:/d/d/c/f/home>su
    root:/>cmd package compile -m speed -f -a
    cmd: Failure calling service package: Failed transaction (2147483646)
    root:/>
**Expected behavior**
It should compile the same way as using ADB to do it.
**Additional information**
Post output of command `termux-info`.
If you are rooted or have access to adb then capture a logcat with `logcat -d "*:W"`, from a adb or root shell.

    u0_a102:/d/d/c/f/home>termux-info
    Updatable packages:
    All packages up to date
    Subscribed repositories:
    https://dl.bintray.com/xeffyr/unstable-packages-24 unstable/main
    https://dl.bintray.com/grimler/termux-root-packages-24 root/stable
    https://dl.bintray.com/termux/termux-packages-24 stable/main
    System information:
    Linux localhost 4.4.100-lineage+ #1 SMP PREEMPT Mon Feb 25 05:22:37 UTC 2019 
    aarch64 Android                                          
    Termux-packages arch:
    aarch64
    Android version:
    8.1.0
    Device manufacturer:
    OnePlus
    Device model:
    ONEPLUS A5000
    u0_a102:/d/d/c/f/home>

**Comment by pthfdr-42** [comment]
Use su twice and it is fixed.

**Comment by robertoprubio** [comment]
I need to execute the "pm disable package/component" from adb (linux environment), but when I execute "adb shell su -c pm disable package/component" I receive the related issue (cmd: Failure calling service package: Failed transaction). I have tried also "adb shell su -c su -c 'pm disable package/component'" but also no luck. Also, I have tried to create a temporary shell script from adb then exec it: "echo 'pm disable package/component > /sdcard/temp.sh && su -c sh /sdcard/temp.sh" but also no luck. Any chance  to get it working from the host PC? Thanks
